n3fit/src/evolven3fit_new/eko_utils.py

```diff
@@ -22,7 +22,7 @@
 }
 
 EVOLVEN3FIT_CONFIGS_DEFAULTS_EXA = {
-    "ev_op_iterations": 10,
+    "ev_op_iterations": 30,
```

Comment:
The runtime factor is roughly x3. Regarding the convergence I think that for short evolution paths 10 is already enough (ev_op_iterations is the number of steps in the numerical evolution). However, for very long ones (for example the last ones in the eko.tar) I think that 30 starts being to small. The problem is that eko allows you to set it once for all so I had to choose a compromise.
I can try increasing it but given that I had problems with 30 I don't know if I will be able to compute it (nikhef cluster has a walltime of 3 days and I go very close to it)

Reply:
No I'm using the smefit nodes. The main problem is that I cannot run with 32 cores otherwise I fill the max memory allowed (512 GB) so I have to use less nodes and in this way it takes almost 3 days

Reply:
Okay let's see. Perhaps it's also possible to ask an admit to (temporally) increase/lift your time limit or ask Emanuele to calculate the eko's for you. I know he does have a lot of computational resources at his disposal but I don't know what his limitations are on a single job. Last effort might be to ask the e-science people since they claim to have a lot of resources as well. 

Anyway, not so relevant for this PR so for as far as I'm concerned it can be merged.